git/password_other.go

```diff
@@ -0,0 +1,16 @@
+//+build !plan9
+
+package git
+
+import (
+	"fmt"
+	"os"
+
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+func getPassword(url string) (string, error) {
```

Comment:
I suppose that in the future we could add platform-specific files for other OSes and use their own password vaults.